deployment/postgres-init-db/sql/tables/merlin/external_events/external_event.sql

```diff
@@ -0,0 +1,77 @@
+-- Create table for external events
+create table merlin.external_event (
+    key text not null,
+    event_type_name text not null,
+	  source_key text not null,
+    derivation_group_name text not null,
```

Comment:
Minor, extra indent:
```suggestion
    event_type_name text not null,
    source_key text not null,
    derivation_group_name text not null,
```